salt/metalk8s/volumes/prepared/files/metalk8s-sparse-volume@.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Setup MetalK8s sparse loop device %I
+RequiresMountsFor=/var/lib/metalk8s/storage/sparse
+
+[Service]
+Type=oneshot
+ExecStart=/sbin/losetup --find --partscan "/var/lib/metalk8s/storage/sparse/%i"
+ExecStop=/usr/local/bin/metalk8s-volume-clean-up "%i"
+TimeoutSec=60
+RemainAfterExit=yes
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
missing newline at the end of the file (as GitHub pointed out ^^)

Reply:
Done.